locales/en-US.json

```diff
@@ -45,19 +45,19 @@
     }
   },
   "needsEslint": {
-    "message": "Add ESLint for code quality?",
+    "message": "Add ESLint for code quality",
```

Comment:
This is no longer a question. Can the `Yes` in the following options be removed? Otherwise, it looks a bit strange that both options contain `Yes`.

And the `negative` option has been removed, the `negative` attribute below can also be removed.